docs/csharp/misc/cs0585.md

```diff
@@ -11,7 +11,7 @@ ms.assetid: e2e23a07-ea37-4dd9-aca6-5714fdbc4ef4
 
 Internal Compiler Error: stage 'stage'
 
- Try to determine if the compiler is failing due to its inability to parse unexpected syntax. If that is not the case, try [Talk to Us](/visualstudio/ide/talk-to-us).
+ Try to determine if the compiler is failing due to its inability to parse unexpected syntax. If that is not the case, try [Talk to Us](/visualstudio/ide/feedback-options).
```

Comment:
Should we look at link text too?

Reply:
That is a fly in the ointment, but the behavior that the customer sees is no different. Even without this change, they'd still click "Talk to Us" and land on "Feedback Options". All this change really does is avoid a redirect hop.

Reply:
I think automating the link text changes would be difficult.

Reply:
Yeah I understand.